### PR TITLE
Adding documentation about SBOM report and how to get access to it

### PR DESCRIPTION
**Pull request by italomg**
**WHAT**
This PR is adding information about the recently implemented SBOM reports. It is using already existing pages to explain what is a SBOM report and how to get access to it.

Resolves https://github.com/aiven/devportal/issues/1540

### CODE REVIEW

File: docs/platform/concepts/cloud-security.rst [hunk 1]
```diff
@@ -77,4 +77,14 @@ Aiven operations team has mandatory recurring training regarding the applicable
...
+
+
+SBOM(Software Bill of Materials)
+--------------------------------
+
+A SBOM is a list of all packages that are being used by Aiven in the services we provide. The list comprehend packages installed in the Node and VM operating system, also the packages installed by the service itself.
+
+SBOMs is being increasingly adopted and might be required for compliance reasons. We provide the report to be download through our CLI as CSV of SPDX files. :doc:`here <docs/tools/cli/project.rst>`
+
+Not all customers are allowed to generate/download SBOMs, if you need an SBOM report from us please reach out to <WHERE?>
```
Comment (lornajane) [inline docs/platform/concepts/cloud-security.rst:90]: I'll mark the PR as draft until all the information is available, please update when you're ready.

File: docs/tools/cli/project.rst [hunk 2]
```diff
@@ -280,3 +280,33 @@ Removes a user with the supplied email address from the project.
...
+  * - ``--project``
+    - The project to fetch details for
+  * - ``--sbom-filepath``
+    - File name, including path and extension (CSV or SPDX), to use
+
+**Example:** Download the SBOM report for the project "my-project", and save it in a file in the current directory called ``sbom.csv``.
+
+::
+
+  avn project generate-sbom --project my-project --sbom-filepath sbom.csv
```
Comment (3clypse) [inline docs/tools/cli/project.rst:312]:
```suggestion
Gets the SBOM report download link for this project, in the chosen output format (CSV or SPDX).

.. list-table::
  :header-rows: 1
  :align: left

  * - Parameter
    - Information
  * - ``--project``
    - The project to fetch details for
  * - ``--output``
    - Output extension (CSV or SPDX), to use

**Example:** Get the SBOM report download link for the project "my-project" in ``csv``.

::

  avn project generate-sbom --project my-project --output csv
```

Comment (3clypse) [reply docs/tools/cli/project.rst:312]: As we modified the `aiven-client` feature + the [temporal external documentation located here](https://aiven.slab.com/posts/sbom-customer-external-doc-rvdyhhsa), suggesting the same changes here too.

**Comment by lornajane** [comment]
If this feature isn't live yet, (it's marked pre-release), then please keep it as draft to prevent accidental merging.

**Comment by jkowall** [comment]
Pending feedback from Jamie to get this pushed through or closed out.

File: docs/tools/cli/project.rst [hunk 3]
```diff
@@ -280,3 +280,33 @@ Removes a user with the supplied email address from the project.
...
+.. list-table::
+  :header-rows: 1
+  :align: left
+
+  * - Parameter
+    - Information
+  * - ``--project``
+    - The project to fetch details for
+  * - ``--output``
+    - Output extension (CSV or SPDX), to use
```
Comment (staceysalamon-aiven) [inline docs/tools/cli/project.rst:306]:
```suggestion
    - Output format (CSV or SPDX)
```

File: docs/tools/cli/project.rst [hunk 4]
```diff
@@ -280,3 +280,33 @@ Removes a user with the supplied email address from the project.
...
+Gets the SBOM report download link for this project, in the chosen output format (CSV or SPDX).
+
+.. list-table::
+  :header-rows: 1
+  :align: left
+
+  * - Parameter
+    - Information
+  * - ``--project``
+    - The project to fetch details for
```
Comment (staceysalamon-aiven) [inline docs/tools/cli/project.rst:304]:
```suggestion
    - The project name
```

File: docs/tools/cli/project.rst [hunk 5]
```diff
@@ -280,3 +280,33 @@ Removes a user with the supplied email address from the project.
...
+  * - ``--output``
+    - Output extension (CSV or SPDX), to use
+
+**Example:** Get the SBOM report download link for the project "my-project" in ``csv``.
```
Comment (staceysalamon-aiven) [inline docs/tools/cli/project.rst:308]:
```suggestion
**Example:** Get the SBOM report download link for the project ``my-project`` in ``csv`` format:
```

**Review by staceysalamon-aiven — CHANGES_REQUESTED** [review]
Thank you for updating this article, @italomg! I've just suggested some minor changes/grammatical fixes. Let me know if there are any issues/questions!

File: docs/platform/concepts/cloud-security.rst [hunk 6]
```diff
@@ -48,11 +48,11 @@ Networking with VPC peering
 
 When using VPC peering, no public internet based access is provided to the services. Service addresses are published in public DNS, but they can only be connected to from the customer's peered VPC using private network addresses.
 
-The service providing virtual machines are still contained under Aiven cloud provider accounts. Aiven VPCs are created per-project and not shared between customers, accounts or projects. When services are deployed in a VPC, the DNS names resolve to their Private IP. 
+The service providing virtual machines are still contained under Aiven cloud provider accounts. Aiven VPCs are created per-project and not shared between customers, accounts or projects. When services are deployed in a VPC, the DNS names resolve to their Private IP.
```
[the remark this answers is not in the document]
Comment (italomg) [reply docs/platform/concepts/cloud-security.rst:51]: yes! Also, should I create a ticket to change `account` appearances in the docs?

File: docs/tools/cli/project.rst [hunk 7]
```diff
@@ -280,3 +280,33 @@ Removes a user with the supplied email address from the project.
 ::
 
   avn project user-remove --project my-project alice@example.com
+
+
+Request project SBOM
+--------------------
+
+SBOM reports are generated per project and can be downloaded as long as the necessary permissions are set for the user/account.
```
[the remark this answers is not in the document]
Comment (italomg) [reply docs/tools/cli/project.rst:288]: actually this is something I realized has changed. This was the initial idea, but the actual implementation check for permission in the project, so I will use `project` instead

File: docs/tools/cli/project.rst [hunk 8]
```diff
@@ -280,3 +280,33 @@ Removes a user with the supplied email address from the project.
...
+
+.. _avn_project_generate_sbom:
+
+``avn project generate-sbom``
+'''''''''''''''''''''''''''''
```
[the remark this answers is not in the document]
Comment (italomg) [reply docs/tools/cli/project.rst:293]: the linter got mad at me when I didn't have that, I will try without it again and check if the linter complains

File: docs/platform/concepts/cloud-security.rst (hunk 6 above)
[the remark this answers is not in the document]
Comment (staceysalamon-aiven) [reply docs/platform/concepts/cloud-security.rst]: They all should have already been removed as part of two PRs merged today for the release, but if you find any others DM me and I'll take care of them!

File: docs/tools/cli/project.rst [hunk 9]
```diff
@@ -276,3 +276,30 @@ Removes a user with the supplied email address from the project.
...
+
+
+Request project SBOM
+--------------------
+
+SBOM reports are generated per project and can be downloaded as long as the necessary permissions are set for the project. You can get the SBOM report download link for a project using the following command:
+
+``avn_project_generate_sbom``
+
+``avn project generate-sbom``
```
Comment (staceysalamon-aiven) [inline docs/tools/cli/project.rst:288]:
```suggestion
```

Comment (staceysalamon-aiven) [reply docs/tools/cli/project.rst:288]: Just deleting the duplicate.

File: docs/tools/cli/project.rst [hunk 10]
```diff
@@ -276,3 +276,30 @@ Removes a user with the supplied email address from the project.
...
 
   avn project user-remove --project my-project alice@example.com
+
+
+Request project SBOM
+--------------------
+
+SBOM reports are generated per project and can be downloaded as long as the necessary permissions are set for the project. You can get the SBOM report download link for a project using the following command:
+
+``avn_project_generate_sbom``
```
Comment (staceysalamon-aiven) [inline docs/tools/cli/project.rst:286]:
```suggestion
``avn_project_generate_sbom``
'''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''

```

Comment (staceysalamon-aiven) [reply docs/tools/cli/project.rst:286]: Hey, I'm sorry, I didn't realize this format was used throughout the article. My bad! I've added the heading line back in.

**Review by staceysalamon-aiven — CHANGES_REQUESTED** [review]
Just reversed my previous suggestion (sorry!) and deleted a duplicate line. Otherwise, looks really good!

File: docs/tools/cli/project.rst (hunk 9 above)
Comment (italomg) [reply docs/tools/cli/project.rst:286]: in that case I think I will keep the other one with the spaces in between words :slightly_smiling_face:

File: docs/tools/cli/project.rst (hunk 10 above)
Comment (italomg) [reply docs/tools/cli/project.rst]: no worries, it is added now :+1:

**Review by staceysalamon-aiven — APPROVED** [review]
LGTM

**Pull request merged**